src/GraphEngine.DataImporter/GraphEngine.DataImporter/Program.cs

```diff
@@ -19,9 +19,32 @@ internal class Program
         static void Main(string[] args)
         {
             CmdOptions opts = new CmdOptions();
-            if (CommandLine.Parser.Default.ParseArguments(args, opts))
+
```

Comment:
revert this. don't hard-code local settings in the program.

Reply:
Yikes - I was testing and forgot to remove testing code.